pm/recipient.go

```diff
@@ -390,6 +391,12 @@ func (r *recipient) redeemWinningTicket(ticket *Ticket, sig []byte, recipientRan
 		return err
 	}
 
+	if monitor.Enabled {
+		// TODO(yondonfu): Handle case where < ticket.FaceValue is actually
+		// redeemed i.e. if sender reserve cannot cover the full ticket.FaceValue
+		monitor.ValueRedeemed(ticket.Sender.String(), ticket.FaceValue)
```

Comment:
`ticket.Sender.String()` will return value in standard ETH address representation - with different case for each letter. This will not match for other addresses that's formatted using `hex.EncodeToString`.
I think this should be lowercased.

Reply:
Good point about being consistent about the address format. I actually think we should use the checksummed (mixed case) ETH address since it is commonly used in applications to check for typos in the address - not particularly important this moment since we're just displaying them in a metrics dashboard, but I think it would be nice to stick with standard convention and it could also be helpful if we're ever copy and pasting addresses from the dashboard into an application that performs checksum validation.

The following commits remove usage of `hex.EncodeToString` in favor of a byte slice -> address -> string conversion:

- 633259a
- 69dbec3